test/images/jessie-dnsutils/BASEIMAGE

```diff
@@ -1,4 +1,4 @@
-amd64=debian:jessie
```

Comment:
rolled back the changes, will look closer later to understand why do we have 2 dnsutil images for same purpose!